save-orchestrator/src/test/kotlin/org/cqfn/save/orchestrator/controller/agents/AgentsControllerTest.kt

```diff
@@ -32,7 +33,8 @@ class AgentsControllerTest {
 
     @Test
     fun checkPostResponseIsOk() {
-        val execution = Execution(3, stubTime, stubTime, ExecutionStatus.PENDING, "stub", "stub").apply {
+        val project = Project("Huawei", "huaweiName", "manual", "huaweiUrl", "description")
```

Comment:
Is this variable used?

Reply:
Yes. execution  must have a non-null project_id

Reply:
Yes, github doesn't highlight change from `2` to `project`, I see it now.